src/frontend/components/DashboardVideoPaneDownloadOption/index.tsx

```diff
@@ -20,42 +24,41 @@ const messages = defineMessages({
 });
 
 interface Props {
```

Comment:
You should also rename this interface for the sake of consistency

Reply:
Good catch 👍